### PR TITLE
Update Roadmap 🚀 

### PR DESCRIPTION
**Pull request by leonardoventurini**

### CODE REVIEW

File: v3-docs/docs/about/roadmap.md [hunk 1]
```diff
@@ -4,81 +4,79 @@ Describes the high-level features and actions for the Meteor project in the near
...
-- Linux ARM Support; ([PR](https://github.com/meteor/meteor/pull/11809))
-- Improve release quality with test coverage and CI automation;
-- Review and help to modernize Meteor tools; ([Discussion](https://github.com/meteor/meteor/discussions/12073))
-- Improve support for Windows 11 or adopt Windows with WSL;
-- Improve Meteor build time; ([Discussion](https://github.com/meteor/meteor/discussions/11587))
-- HTTP/3 Support;
-- Tree-shaking; ([PR](https://github.com/meteor/meteor/pull/11164))
-- Support package.json exports fields; ([Discussion](https://github.com/meteor/meteor/discussions/11727))
+- Bringing community packages to the core
+> Some packages are widely used and should be part of the core, so this involves identifying and moving them there.
```
Comment (henriquealbert) [inline v3-docs/docs/about/roadmap.md]:
```suggestion
> Some packages are widely used and should be part of the core, which involves identifying and moving them to the core.
```

File: v3-docs/docs/about/roadmap.md [hunk 2]
```diff
@@ -4,81 +4,79 @@ Describes the high-level features and actions for the Meteor project in the near
...
 
-### Candidate items
-We need to discuss further to decide whether or not to proceed with these implementations.
+- MongoDB Change Streams support ([Discussion](https://github.com/meteor/meteor/discussions/11842))
+> Change Streams is the official way to listen to changes in MongoDB; we should provide a way to use it in Meteor, seamlessly. It has been planned for a long time, and now we’re in a position to do it.
```
Comment (henriquealbert) [inline v3-docs/docs/about/roadmap.md]:
```suggestion
> Change Streams is the official way to listen to changes in MongoDB. We should provide a way to use it seamlessly in Meteor. It has been planned for a long time, and now we’re in a position to do it.
```

**Review by henriquealbert — DISMISSED** [review]
lgtm

File: v3-docs/docs/about/roadmap.md [hunk 3]
```diff
@@ -4,81 +4,79 @@ Describes the high-level features and actions for the Meteor project in the near
...
-- Remove Fibers entirely;
-- Update Cordova integration to Meteor 3.0;
-- Run Meteor on Node.js 18;
-- Change web engine from Connect to Express;
+- Improve TypeScript support for Meteor and packages ([Discussion](https://github.com/meteor/meteor/discussions/12080))
+> Should be an ongoing effort to improve the TypeScript support in Meteor and packages, as we write new code in TypeScript and get more skin in the game, it should naturally improve. This doesn’t mean we will actively refactor working code.
 
-### Next releases
+- Performance improvements for Meteor 3.0
+> After the removal of fibers, we became heavily reliant on async resources and consequently Async Hooks/Async Local Storage, that has a performance cost, we need to optimize that.
```
Comment (fredmaiaarantes) [inline v3-docs/docs/about/roadmap.md]:
```suggestion
> After removing fibers, we became heavily reliant on async resources and consequently Async Hooks/Async Local Storage, which has a performance cost, we need to optimize that.
```

File: v3-docs/docs/about/roadmap.md [hunk 4]
```diff
@@ -4,81 +4,79 @@ Describes the high-level features and actions for the Meteor project in the near
...
+- Bringing community packages to the core
+> Some packages are widely used and should be part of the core, so this involves identifying and moving them there.
 
-### Candidate items
-We need to discuss further to decide whether or not to proceed with these implementations.
+- MongoDB Change Streams support ([Discussion](https://github.com/meteor/meteor/discussions/11842))
+> Change Streams is the official way to listen to changes in MongoDB; we should provide a way to use it in Meteor, seamlessly. It has been planned for a long time, and now we’re in a position to do it.
+
+- Integrated support for Vite (Client Bundler, [Discussion](https://github.com/meteor/meteor/discussions/11587))
+> Vite is a fast and modern client bundler with an amazing ecosystem, it has many potential benefits for Meteor: build performance, tree-shaking, making our codebase leaner, and focus on what we do best.
```
Comment (fredmaiaarantes) [inline v3-docs/docs/about/roadmap.md]:
```suggestion
> Vite is a fast and modern client bundler with an amazing ecosystem. It has many potential benefits for Meteor: build performance, tree-shaking, making our codebase leaner, and focusing on what we do best.
```

File: v3-docs/docs/about/roadmap.md [hunk 5]
```diff
@@ -4,81 +4,79 @@ Describes the high-level features and actions for the Meteor project in the near
...
+> Sometimes our CI/CD takes too long to run, causing long queues and delays in our release process and feedback loop, we need to improve that.
+
+- Improve support for Windows 11
+> We had many complaints in the past, we need to research and make architectural improvements to make it easier to support Windows, not just punctual fixes.
+
+- Document better Windows with WSL
+> It's already possible to use Meteor on Windows with WSL, but we need to document it better
+
+- HTTP/3 Support
+> HTTP/3 is the next version of the HTTP protocol, we should support it in Meteor to leverage its performance and security benefits.
```
Comment (fredmaiaarantes) [inline v3-docs/docs/about/roadmap.md]:
```suggestion
> HTTP/3 is the next version of the HTTP protocol. We should support it in Meteor to leverage its performance and security benefits.
```

File: v3-docs/docs/about/roadmap.md [hunk 6]
```diff
@@ -4,81 +4,79 @@ Describes the high-level features and actions for the Meteor project in the near
...
+- Remove Fibers dependency from Meteor Public APIs
+- Remove Fibers entirely
+- Update Cordova integration to Meteor 3.0
+- Run Meteor on Node.js v20
+- Change web engine from Connect to Express
 
 -----------
 
-For more completed items, refer to our [changelog](https://docs.meteor.com/changelog.html).
+For more completed items, refer to our [changelog](https://docs.meteor.com/about/roadmap.html).
```
Comment (fredmaiaarantes) [inline v3-docs/docs/about/roadmap.md]:
```suggestion
For more completed items, refer to our [changelog](https://docs.meteor.com/history.html).
```

**Review by fredmaiaarantes — APPROVED** [review]

**Comment by harryadel** [comment]
Great work :clap:

**Review by nachocodoner — APPROVED** [review]

**Pull request merged**